HW27/HomeWork27.cpp

```diff
@@ -0,0 +1,19 @@
+#include "HomeWork27.h"
+
+VectorRelativeState Vector2d::getRelativeState(const Vector2d& other) const
+{
+    float dot = this->dotProduct(other);
+    float length1 = std::sqrt(x * x + y * y);
+    float length2 = std::sqrt(other.x * other.x + other.y * other.y);
+    float cosTheta = dot / (length1 * length2);
```

Comment:
can be calculated without cosine calculations. See Lecture slides for reference how to determine that using only dotProduct value